Makefile

```diff
@@ -68,4 +68,7 @@ license:
 docs:
 	cargo doc --no-deps --all-features
 
+run:
+	cargo run -p forest -- --config config.toml	
```

Comment:
was this just meant as a temp for testing? Seems weird to use a config file that doesn't exist by default

Reply:
Yeah I will remove